translations/es.json

```diff
@@ -0,0 +1,210 @@
+    "unsupported": "no soportado",
+    "disconnect": "Desconectar",
+    "error": "No se pudieron cargar los datos",
+    "token": "Simbólico",
+    "amount": "Monto",
+    "deposit": "Depósito",
+    "withdraw": "Retirar",
+    "pause": "Pausa",
+    "resume": "Resumir",
+    "or": "o",
+    "name": "Nombre",
+    "address": "Dirección",
+    "history": "Historia",
+    "edit": "Editar",
+    "update": "Actualizar",
+    "month": "Mes",
+    "year": "Año",
+    "close": "Cerrar",
+    "goBack": "Regresa",
+    "menu": "Menú",
+    "streamsAndHistory": "Corrientes e Historia",
+    "logoAlt": "Logotipo de {nombre}",
+    "transactionSubmitted": "Transacción enviada",
+    "viewOnExplorer": "Ver en {explorador}",
+    "eventTimestamp": "Marca de tiempo del evento",
+    "moreInfo": "Más información",
+    "oldStream": "Viejo arroyo",
+    "newStream": "Nueva corriente",
+    "stream": "Arroyo",
+    "payer": "Pagador",
+    "payee": "Tenedor",
+    "streamHistory": "Historial de transmisión",
+    "eventType": "Tipo de evento",
+    "timestamp": "marca de tiempo",
+    "you": "Tú"
+  },
+  "Header": {
+    "connectWallet": "Conectar billetera",
+    "language": "Idioma",
+    "docs": "Documentos"
+  },
+  "Banner": {
+    "heading": "¡Automatice los pagos de salarios recurrentes en criptografía!",
+    "description": "Automatice los txs de salarios, transmitiéndolos por segundo para que los empleados puedan retirarlos cuando lo deseen y no tenga que lidiar con el envío de txs manualmente.",
+    "close": "Cerrar pancarta"
+  },
+  "Balances": {
+    "heading": "Saldos",
+    "balance": "Equilibrio",
+    "connectWallet": "Conecte la billetera para ver sus saldos",
+    "deposit": "Depositar nuevo token",
+    "noData": "Crear un saldo primero",
+    "error": "No se pudieron cargar saldos",
+    "toDepleted": "Agotado",
+    "monthlyCost": "Costo mensual",
+    "topup": "Completar",
+    "streamsDepleted": "Corrientes agotadas",
+    "noStreams": "Sin transmisiones",
+    "days": "dias"
+  },
+  "Streams": {
+    "heading": "Corrientes",
+    "activeStreams": "Corrientes activas",
+    "connectWallet": "Conecte la billetera para ver sus transmisiones",
+    "create": "Crear transmisión",
+    "error": "No se pudieron cargar transmisiones",
+    "noData": "Cree una secuencia para ver una lista de sus secuencias creadas aquí",
+    "noActiveStreams": "No hay transmisiones activas",
+    "totalStreamed": "Transmitido total",
+    "withdrawable": "extraíble",
+    "send": "Enviar",
+    "modify": "Modificar",
+    "paused": "en pausa",
+    "cancel": "Cancelar",
+    "outOfFunds": "sin fondos",
+    "outgoingStream": "Corriente saliente",
+    "incomingStream": "Corriente entrante",
+    "incomingStreamFrom": "Transmisión entrante de",
+    "disperse": "Dispersar",
+    "withdrawAnotherWallet": "Retirar otra billetera",
+    "sendAll": "Envia todo",
+    "editName": "Editar el nombre de la dirección del beneficiario"

```

Comment:
Editar el nombre del beneficiario

Reply:
I think this one is fine, can you review? I have been closing without comments the one I agree with. 
Also thank you so much!